docs/usage/self-hosted-experimental.md

```diff
@@ -19,11 +19,6 @@ We will try to keep breakage to a minimum, but make no guarantees that an experi
 If set, Renovate will export OpenTelemetry data to the supplied endpoint.
 For more information see [the OpenTelemetry docs](opentelemetry.md).
 
-## `RENOVATE_EXPERIMENTAL_NO_MAVEN_POM_CHECK`
-
-If set to any value, Renovate will skip its default artifacts filter check in the Maven datasource.
-Skipping the check will speed things up, but may result in versions being returned which don't properly exist on the server.
-
 ## `RENOVATE_PAGINATE_ALL`
```

Comment:
Any reason why this is migrated?

Reply:
It was in the experimental file list so I assumed that it shuold be converted as well.